README.md

```diff
@@ -189,13 +189,24 @@ Authors should respect the `prefers-reduced-motion` media query by conditionally
+
+We want the history model for portals to conform to users' expectations of the back button. Specifically, the back button should take them back to the last thing they saw. Unfortunately, as mentioned [below](#summary-of-differences-between-portals-and-iframes), the mental model of a portal may differ between users and web developers. While developers will see portals as separate contexts navigating independently from their host context, users will not. This means that the portal context should not have an independent session history which we aggregate with its host context to present to the user. To illustrate how this could cause problems, consider `example.com/a.html` loading `b.html` in a portal, then navigating the portal to `c.html`, then activating the portal. If the user pressed the back button, it would not be appropriate to load `b.html`, since (1) this makes for bad UX as this is a state the user has not seen before and, (2) this is a concern for privacy as the user's browsing history would contain a page they did not intend to visit. Even for states that the user has seen previously, having a single navigation reintroduce potentially multiple history states maps poorly to the web's chronological history model, as opposed to the mental models for tab switching or mobile device style multitasking.
+
+Navigations within portals are subject to certain security restrictions. Notably, content that does not allow embedding (e.g. via `X-Frame-Options` [see above](#permissions-and-policies)) cannot load in a portal. Furthermore, both the portal host and the portal content must use an HTTP(S) scheme. This is in order to establish the origin of the content for it to be used safely.
+
+Navigation errors within portals may cause portal activation to be rejected. Instead of, for example, the user agent showing an error page to the user as with a conventional navigation, the promise returned by the activate method allows a page to gracefully handle the rejection. Furthermore, user agents have existing limitations on navigations initiated by the page where they may be ignored if they are considered to conflict with a user's intent to perform a different navigation. Such cases are not described by the existing navigation spec (see [#218](https://github.com/WICG/portals/issues/218)), but portal activations are subject to these limitations. In the case where another navigation takes precedence over portal activation, the promise returned by the activate method rejects.
```

Comment:
> Navigation errors within portals may cause portal activation to be rejected.

I think that may be exposing something we don't currently expose.

Reply:
Worth filing an issue and pointing to it from here, perhaps.

Reply:
The same information would be exposed in other ways. For example, |fetch('https://doesnotexist.example.com/')| rejects.

Reply:
But you can't tell if that fetch fails because of lack of CORS headers, or because of an actual network error.

Reply:
I think there are cases where that isn't true. `X-frame-options: DENY` wouldn't cause fetch to fail, but it would cause a portal to fail, for instance.

Reply:
Oops, I missed issue #228 which talks about this. I'll reference it here for now.